db_test.go

```diff
@@ -425,6 +425,40 @@ func (s *SurrealDBTestSuite) TestSmartMarshalQuery() {
 	})
 }
 
+// BenchmarkCreate benchmarks the creation of a record
+func BenchmarkCreate(b *testing.B) {
+	s := new(SurrealDBTestSuite)
+	s.SetT(&testing.T{})
+	s.SetupSuite()
```

Comment:
You wouldnt want to use a real database in this scenario because then you have network overhead and other issues involved in the benchmark. Perhaps instead use mocks? Gomock may be a good option for that. The create and other functions should be performance tested for parsing parameters and deserialising structs.

Reply:
done